### PR TITLE
[11.0][ADD] sale_order_archive

### PR DESCRIPTION
**Pull request by astirpe**
On a system with a high volume of sales, the number of sale orders displayed in the list view can become huge. This module allows to archive Sale Orders that are in status Locked or Cancelled.

### CODE REVIEW

**Comment by astirpe** [comment]
@CasVissers

**Review by tafaRU — APPROVED** [review]
:+1:  Technical and functional review.

**Review by bodedra — APPROVED** [review]
LGTM :+1:

File: sale_order_archive/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+On a system with a high volume of sales, the number of sale orders displayed in the list view can become huge.
+This module allows to archive Sale Orders that are in status Locked or Cancelled.
+
+If a sale order is archived, it will be hidden from the sale orders list view.
+
+This module only depends on module sale, but it could be used in combination with OCA module 'record_archiver' in order
+to automatically archive old sale orders.
```
Comment (thomaspaulb) [inline sale_order_archive/README.rst:34]: Could you add a link to this? Am only seeing that module for v7.0 - or is this a roadmap item?

**Review by thomaspaulb — APPROVED** [review]

File: sale_order_archive/README.rst (hunk 1 above)
Comment (astirpe) [reply sale_order_archive/README.rst:34]: Is not yet ported to versions higher that 7.0 AFAIK. Not sure that the link to a V7.0 would be an help.
This paragraph was meant to just explain the potentials of this module.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**